src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3363,6 +3406,46 @@ components:
           type: string
           example:
             "Main group"
+    GetClearingHistory:
+      type: object
+      properties:
+        date:
+          type: string
+          format: YYYY-mm-ddTHH:MM:SS
+          description: Date of the clearing history
+          example: "2023-06-15"
+        username:
+          type: string
+          description: Username of the user who created the decision.
+          example: "fossy"
+        scope:
+          type: string
+          description: Scope of the clearing
+          enum:
+            - local
+            - package
+            - global
+        type:
+          type: string
+          description: Type of the clearing
+          enum:
+            - TO_BE_DISCUSSED
+            - IRRELEVANT
+            - IDENTIFIED
+            - DO_NOT_USE
+            - NON_FUNCTIONAL
```

Comment:
I think this will update as well according to

https://github.com/fossology/fossology/blob/b42626c170f7eac68de7e246683646b2aa00afbf/src/lib/php/Data/DecisionTypes.php#L23-L29

Reply:
Updated, please let me know if there's anything remaining @GMishx